rules/ruby/lang/eval_linter.yml

```diff
@@ -0,0 +1,40 @@
+patterns:
+  - eval()
```

Comment:
We could make this something like `eval($<NOT_USER_INPUT>$<...>)` (using `ruby_shared_common_user_input`), so we don't trigger on the same cases as the other rule.

Reply:
Good shout 🙏